<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>scurve auto plot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>scurve control</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_feed_override">
              <property name="maximum">
               <double>150.000000000000000</double>
              </property>
              <property name="value">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>feed override</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_maxvel">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>maxvel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_setall">
              <property name="text">
               <string>set all</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>maxjerk</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_maxjer">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>maxacc</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_maxacc">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>adaptive feed</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_adafee">
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="value">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_pause">
              <property name="text">
               <string>pause</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_resume">
              <property name="text">
               <string>resume</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_run">
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_revmot">
              <property name="text">
               <string>Reverse motion</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_pause_timer">
              <property name="text">
               <string>pause timer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>scurve status</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>curvel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="label_segpro">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>lenght end</string>
              </property>
             </widget>
            </item>
            <item row="8" column="4">
             <widget class="QLabel" name="label_seg_ve">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_curvel">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="7" column="4">
             <widget class="QLabel" name="label_seg_vo">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>pause</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_curacc">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="label_segbeg">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>traject progress</string>
              </property>
             </widget>
            </item>
            <item row="8" column="5">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>mm/s</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>curpos</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_trjpro">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="6" column="4">
             <widget class="QCheckBox" name="checkBox_revmot_status">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>curacc</string>
              </property>
             </widget>
            </item>
            <item row="8" column="3">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>segment ve</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_vectnr">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QCheckBox" name="checkBox_pause">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>traject lenght</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>vector nr</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>lenght beg</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QCheckBox" name="checkBox_finished">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>traject finished</string>
              </property>
             </widget>
            </item>
            <item row="7" column="5">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>mm/s</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>segment progress</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_curpos">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>segment vo</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>mm/s</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="label_segend">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>mm/s^2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_trjlen">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>reverse motion</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>pause timer</string>
              </property>
             </widget>
            </item>
            <item row="5" column="4">
             <widget class="QCheckBox" name="checkBox_pause_timer">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
